.gitlab/docker/Dockerfile.spack_dependencies

```diff
@@ -1,6 +1,10 @@
 ARG BASE_IMAGE
 FROM $BASE_IMAGE
 
+# Tmp! Modify the meson package.py to find the binary
+RUN sed -i "s/self.spec.prefix.bin.meson/self.spec.prefix.local.bin.meson/" \
+    $SPACK_ROOT/var/spack/repos/builtin/packages/meson/package.py
+
```

Comment:
See other comment about not using the system python. I think if that works you should be able to remove this workaround.